src/main/java/com/altenheim/kalender/controller/logicController/SystemNotificationsController.java

```diff
@@ -0,0 +1,106 @@
+package com.altenheim.kalender.controller.logicController;
+
+import java.awt.*;
+import java.awt.event.*;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.awt.TrayIcon.MessageType;
+
+import com.altenheim.kalender.interfaces.ICalendarEntriesModel;
+import com.altenheim.kalender.interfaces.ISystemNotificationsController;
+import com.altenheim.kalender.models.SettingsModel;
+import com.calendarfx.model.Entry;
+
+public class SystemNotificationsController extends TimerTask implements ISystemNotificationsController  
+{
+    private SettingsModel settings;
+    private ICalendarEntriesModel administrateEntries;
+
+    private SystemTray tray = SystemTray.getSystemTray();
```

Comment:
Du hast ja unten ne schöne Methode gebastelt für die Initialisierung. Nun rufst du aber hier (gleich nachdem du den Konstruktor aufrufst) diese Methoden VOR der Initialisierung auf. Mal abgesehen davon, dass das die ganze Struktur obsolet macht ist es gar nicht nötig - du verwendest ja tray außer bei der Init nirgends. Also warum hier ein globales Objekt???

Reply:
Global, damit auch andere Methoden in der Klasse drauf zugreifen können

Reply:
Und welche? Sehe da keine. Zudem, selbst wenn es so wäre: challange: wo sollte das dann initialisiert werden?

Reply:
ja stimmt hatte ich wieder raus genommen, hab jetzt die Initialisierung richtig gemacht

Reply:
Warum ist es denn immer noch hier als globale property? :-/